LICENSE

```diff
@@ -1,6 +1,7 @@
 BSD 3-Clause License
 
 Copyright (c) 2018, Redis Labs Modules
+Portions Copyright (c) 2018, Red Hat
```

Comment:
please remove this line

Reply:
The code in this contribution was authored by Red Hat, and copyright is retained by Red Hat - as discussed earlier in https://github.com/RedisLabsModules/RLTest/pull/13

Reply:
@gkorland can you explain the issue you see with sharing copyright with others?  Perhaps there's a misunderstanding there we can work through.

Reply:
As for your request we removed the CLA requirement but we're not going to add to the License file a copyright notice for every PR we get.

Reply:
@gkorland ah OK, I see - I think I understand your concern there.  In practical terms, and from my experience in other projects I've been involved with, I would suggest its unlikely that every PR will request an addition to the LICENSE file - very few will.  When it does happen, it's something to celebrate - the project is getting noticed and the community is getting involved - these are good things and should be encouraged.  Less work for you!  8^)

This code contribution has been funded by Red Hat, and working with python3 its a useful and important step forward for the project in the long term ... so I (respectfully, peacefully) request that Red Hat's copyright on this contribution be acknowledged as requested.  Thanks!